.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
--   repo: git://github.com/pre-commit/pre-commit-hooks
+-   repo: git@github.com:pre-commit/pre-commit-hooks
```

Comment:
nit: it won't be simpler to use https:// instead? The other two are already based on https. I like `https` since i can easily test if the git repository is accessible by clicking on it.